spec.emu

```diff
@@ -234,6 +234,81 @@ contributors: Google, Ecma International
     <emu-note>
       The content in this table is proposed by Era Code WG to CLDR TC in <a href="https://github.com/unicode-org/cldr/pull/2665">PR 2665</a> and under review. It is targeted to be part of CLDR 43, scheduled to be released in April 2023.
     </emu-note>
+    <emu-table id="table-epoch-years">
+      <emu-caption>epoch years</emu-caption>
+      <table class="real-table">
+        <thead>
+          <tr>
+            <th>Calendar</th>
+            <th>Epoch ISO Year</th>
+          </tr>
+        </thead>
+        <tr>
+          <td>*"buddhist"*</td>
+          <td>-543</td>
+        </tr>
+        <tr>
+          <td>*"chinese"*</td>
+          <td>0</td>
+        </tr>
+        <tr>
+          <td>*"coptic"*</td>
+          <td>284</td>
```

Comment:
I'm not sure about this, but should be 283?
```js
> new Date('000283-08-30').toLocaleString('en-ca-u-ca-coptic', { dateStyle: 'short' })
'1/1/1 ERA0'
```
(ERA0 = the "BD" era which ICU removed, the following year is 1 ERA1. So 1 ERA0 = arithmetic year 0, and 1 ERA0 starts in ISO year 283)